src/app/domains/wallets/domains.wallets.types.ts

```diff
@@ -23,7 +23,18 @@ export interface IWalletTransactionApiResponse {
   creationDate: TServerDateTime;
 }
 
+export interface IWalletTransactionItemData extends Omit<IWalletTransactionApiResponse, 'id'> {
```

Comment:
Maybe to make all of this a bit cleaner lets move all transaction related stuff from `domains/wallets` to `domains/transactions` including types, gateways, mocks etc